kuma/wiki/tasks.py

```diff
@@ -202,6 +203,18 @@ def delete_old_revision_ips(immediate=False, days=30):
 
 
 @task
-def send_first_edit_email(email):
-    email.to = [config.EMAIL_LIST_FOR_FIRST_EDITS,]
+def send_first_edit_email(revision_pk):
+    """ Make an 'edited' notification email for first-time editors """
+    revision = Revision.objects.get(pk=revision_pk)
+    user, doc = revision.creator, revision.document
+    subject = ("[MDN] %(user)s made their first edit, to: %(doc)s" %
+               {'user': user.username, 'doc': doc.title})
+    message = render_to_string('wiki/email/edited.ltxt',
+                               context_dict(revision))
+    current_site = Site.objects.get_current()
+    doc_url = "https://%s%s" % (current_site.domain, doc.get_absolute_url())
```

Comment:
Surprised there's no helper for this. It seems like you'd do this a lot? In zamboni we have `absolutify` which is used in both Python and template code.